app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserFragment.java

```diff
@@ -269,6 +274,11 @@ private void updateMenuItems() {
             _fragmentMenu.findItem(R.id.action_create_shortcut).setVisible(selMulti1 && (selFilesOnly || selDirectoriesOnly));
             _fragmentMenu.findItem(R.id.action_check_all).setVisible(_filesystemViewerAdapter.areItemsSelected() && selCount < totalCount);
             _fragmentMenu.findItem(R.id.action_clear_selection).setVisible(_filesystemViewerAdapter.areItemsSelected());
+
+            final MenuItem sortItem = _fragmentMenu.findItem(R.id.action_sort);
+            if (sortItem != null) {
+                _cu.tintDrawable(sortItem.getIcon(), _dopt.sortOrder.isFolderLocal ? GsFileBrowserListAdapter.FAVOURITE_COLOR : Color.WHITE);
```

Comment:
Highlight the sort icon when the folder has a custom sort. This is a little visual hint